### ISSUE TITLE
No option to show treasure

### ISSUE THREAD

**Issue by Jon-Murray** [IrcDirk/Carbonite-Classic#45]
The gathermate2 data for treasure is present. However, i cannot find a way to display the minimap icons in the same way as import/show herbs and mining.

Thanks for the great addon.

**Comment by IrcDirk** [comment]
Yes. This was not implemented before (even in Retail). So this is feature request. Some things need to be fixed first then ill add some features.
